internal/dataexchange/ffdx/dxevent.go

```diff
@@ -154,8 +156,14 @@ func (h *FFDX) dispatchEvent(msg *wsEvent) {
 		var hash *fftypes.Bytes32
 		hash, err = fftypes.ParseBytes32(h.ctx, msg.Hash)
 		if err == nil {
+			var ns string
+			pathParts := strings.Split(msg.Path, "/")
+			if len(pathParts) >= 2 {
+				ns = pathParts[len(pathParts)-2]
```

Comment:
This seems to work, but feels a bit fragile. I wonder if namespaces should be more explicitly included on dx blob events?

Reply:
It's a good point, in-line with the work on other microservice connectors to make them namespace aware at the API boundary.